dimod/quadratic/quadratic_model.py

```diff
@@ -342,16 +354,31 @@ def shape(self) -> Tuple[int, int]:
 
     @property
     def variables(self) -> Variables:
-        """The variables of the quadratic model"""
+        """The variables of the quadratic model.
+
+        Examples:
+
+            >>> qm = dimod.QuadraticModel()
+            >>> qm.add_variable('INTEGER', 'i')
+            'i'
+            >>> qm.add_variable('BINARY')
+            1
+            >>> qm.add_variable('BINARY', 'y')
+            'y'
+            >>> qm.variables
+            Variables(['i', 1, 'y'])
+
+        """
         return self.data.variables
 
     @forwarding_method
     def add_linear(self, v: Variable, bias: Bias):
-        """Add a quadratic term."""
+        """Add a linear bias to an existing variable."""
         return self.data.add_linear
 
     @forwarding_method
     def add_quadratic(self, u: Variable, v: Variable, bias: Bias):
+        """Add quadratic bias to an interaction between two variables."""
```

Comment:
"Add quadratic bias to an interaction between two variables." should be, "Add quadratic bias as an interaction between two variables" or "Add quadratic bias to a pair of variables."